weather_mv/loader_pipeline/bq_test.py

```diff
@@ -370,50 +370,18 @@ def test_multiple_editions(self):
         self.test_data_path = f'{self.test_data_folder}/test_data_grib_multiple_edition_single_timestep.bz2'
         actual = next(self.extract(self.test_data_path))
         expected = {
```

Comment:
Ideally, each row would have all available variables for each lat/lng/time coordinate. Is that possible?